hwbench/environment/block_devices.py

```diff
@@ -69,6 +75,15 @@ def get_udev_attributes(self) -> dict[str, Any]:
             )
         return dumped
 
+    def get_manufacturer(self) -> str:
+        if not self.wwn:
+            return "000000"
+
+        ouidb = OUI()
+        self.oui = ouidb.wwn_to_oui(self.wwn)
+        self.manufacturer = ouidb.hex_to_manufacturer(self.oui.upper())
```

Comment:
You don't need to upper oui as the hex_to_manufacturer already do it.

Reply:
done